### DISCUSSION TITLE
Could the scripted install be updated to allow for downloading a given version, not just most recent?

### DISCUSSION THREAD

**Discussion by jwebber1** [slimtoolkit/slim#425] [Ideas]
Could the scripted install from the Readme...

```
Scripted Install

You can also use this script to install the current release of DockerSlim on Linux (x86 and ARM) and macOS (x86 and Apple Silicon)

curl -sL https://raw.githubusercontent.com/docker-slim/docker-slim/master/scripts/install-dockerslim.sh | sudo -E bash -
```

... be updated to allow for installation of a specific version? For example, instead of installing 1.39.0, I may want to install 1.37.5 specifically.

**Comment by jwebber1** [comment]
I guess we are good to close this.

**Comment by kcq** [comment]
there's a quick work around for this... the script dynamically sets the `VER` env variable based on the latest version, but you can comment it out and set `VER` to the version you want. the next release will included an updated version that will allow you to specify the version explicitly

**Comment by kcq** [comment]
The `scripts/install-slim.sh` helper script has been enhanced to accept an optional release version. It's in 'master' if you want to check it out.
